src/aap_eda/analytics/analytics_collectors.py

```diff
@@ -478,6 +478,57 @@ def event_streams_by_activation_table(
     return _copy_table("event_streams_by_activation", query, full_path)
 
 
+@register(
+    "event_streams_by_running_activations_table",
+    "1.0",
+    format="csv",
+    description="Data on event_streams used by each running activation",
+)
+def event_streams_by_running_activations_table(
+    since: datetime, full_path: str, until: datetime, **kwargs
+):
+    running_states = [
+        ActivationStatus.RUNNING,
+        ActivationStatus.STARTING,
+        ActivationStatus.PENDING,
+    ]
+    activations = (
+        models.Activation.objects.filter(
+            Q(created_at__gt=since, created_at__lte=until)
+            | Q(modified_at__gt=since, modified_at__lte=until)
+        )
+        .filter(status__in=running_states)
+        .distinct()
+    )
+
+    event_streams = models.EventStream.objects.none()
+    for activation in activations:
+        event_streams |= activation.event_streams.all()
+
+    if not bool(event_streams):
+        return
+
+    event_streams = event_streams.annotate(
+        event_stream_id=F("id"),
+        activation_id=F("activations__id"),
```

Comment:
This query will duplicate the event stream record per activation related if there are multiple activations using the same event stream. Is this the desired outcome or maybe we want only one record per event stream with a list of the activation ids consuming it? 

I don't have context about this data will be consumed, I'm not saying it is wrong as it is, I'm just making sure.

Reply:
Yeah, the analytics team can easily use the flat list with other data to combine the final result.